.github/workflows/integration_test.yml

```diff
@@ -67,11 +67,16 @@ on:
         required: false
         default: true
         type: boolean
-      test-goal:
-        description: The Maven goal(s) used by test jobs
+      test-goal-java:
```

Comment:
Don't forget to update all places where we use this! 😅

Reply:
Yup, this is only used in the `invgrp-demo` plugin, I'll update it.